@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,126 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef, useEffect, useState } from "react";
+
+interface ProgressBarPropsBase
+  extends Omit<HTMLAttributes<HTMLDivElement>, "role"> {
+  /**
+   * Changes height.
+   * @default "medium"
+   */
+  size?: "large" | "medium" | "small";
+  /**
+   * Current progress. When duration is set, value is ignored.
+   */
+  value: number;
+  /**
+   * Maximum progress.
+   */
+  valueMax: number;
+  /**
+   * Expected task duration in seconds.
+   * ProgressBar grows with a preset animation for {duration} seconds
+   * Afterwards, it shows an indeterminate animation.
+   */
+  duration?: number;
+  /**
+   * String ID of the element that labels the progress bar.
+   * Not needed if `aria-label` is used.
+   */
+  "aria-labelledby"?: string;
+  /**
+   * String value that labels the progress bar.
+   * Not needed if `aria-labelledby` is used.
+   */
+  "aria-label"?: string;
+}
+
+export type ProgressBarProps = ProgressBarPropsBase &
+  (
+    | {
+        "aria-labelledby": string;
+        "aria-label"?: never;
+      }
+    | {
+        "aria-label": string;
+        "aria-labelledby"?: never;
+      }
+  );
+
+/**
+ * ProgressBar
+ * - Visualizes how far along the user is in a process.
+ *
+ * @see [📝 Documentation](https://aksel.nav.no/komponenter/core/progress-bar)
+ * @see 🏷️ {@link ProgressBarProps}
+ *
+ * @example
+ * // For loading content
+ * <ProgressBar value={20} valueMax={100} duration={12} />
```

Comment:
Både value og valueMax blir vel irrelevant ved bruk av duration? 🤔
```suggestion
 * <ProgressBar duration={12} />
```